org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/functions/FunctionComposer.java

```diff
@@ -128,4 +130,16 @@ public FunctionComposer(IFunctionFactory functionFactory, @Assisted IProject pro
         return parseFunction()
             .kleisliCompose(functionFactory.createStyleFunction(project, lang));
     }
+
+    /**
+     * Composes a {@link PrettyPrintFunction}, which provides pretty printing.
+     *
+     * @return {@link PrettyPrintFunction} - The pretty print function.
+     */
+    public FailableFunction<ISpoofaxTermResult<?>, PrintResult, IResult> pPrettyPrintFunction() {
```

Comment:
I wouldn't go with the `p` prefix. It is used by other functions to denote whether they were chained to analyze or parse, but in this case it is evident.

Possibly even consider prefixing with `term` to denote that it requires an `ISpoofaxTermResult<?>`